great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -785,3 +790,138 @@ def _build_categorical_rule() -> Rule:
         )
 
         return rule
+
+    @staticmethod
+    def _build_text_rule() -> Rule:
+
+        # Step-1: Instantiate "ColumnDomainBuilder" for selecting proper text columns.
+
+        text_column_type_domain_builder: ColumnDomainBuilder = ColumnDomainBuilder(
+            include_column_names=None,
+            exclude_column_names=None,
+            include_column_name_suffixes=None,
+            exclude_column_name_suffixes=None,
+            semantic_type_filter_module_name=None,
+            semantic_type_filter_class_name=None,
+            include_semantic_types=[
+                SemanticDomainTypes.TEXT,
+            ],
+            exclude_semantic_types=None,
+            data_context=None,
+        )
+
+        # Step-2: Declare "ParameterBuilder" for every metric of interest.
+
+        column_min_length_range_estimator_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(
```

Comment:
@cdkini For this, there is now a common method in `DataAssistant` -- create the `NumericMetricRangeMultiBatchParameterBuilder` for you.  Usage example:
```
        column_min_values_range_parameter_builder_for_validations: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_numeric_metric_range_multi_batch_parameter_builder(
            metric_name="column.min",
            metric_value_kwargs=None,
            json_serialize=True,
        )
```

Reply:
@cdkini However, please notice that the metrics parameter builders probably never have the case of needing a range.  Please check out the numeric and/or datetime rule implementations for the pattern.  Thanks!

Reply:
sounds good, will look now. Thanks!